jenkins/jobs/bmo_e2e_tests.pipeline

```diff
@@ -0,0 +1,89 @@
+import java.text.SimpleDateFormat
```

Comment:
May be the name of the file should have optional in it as well?

Reply:
@Sunnatillo this is still open